storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -230,6 +235,55 @@ public Optional<String> serverConfigName(String configName) {
         FOLLOWER_REPLICATION_THROTTLED_REPLICAS_CONFIG
     ));
 
+    private static final String DEFAULT_LOG_DIR = "/tmp/kafka-logs";
```

Comment:
This class is for configurations for the storage layer. We should not put unrelated configurations like `ZOOKEEPER_CONNECT_PROP` in it.

Reply:
@mimaison scala version of StorageTool executes this code block https://github.com/apache/kafka/blob/4981fa939d588645401619bfc3e321dc523d10e7/core/src/main/scala/kafka/server/KafkaConfig.scala#L2181C5-L2181C25 Hence I added similar in LogConfig. wdyt ?